.github/workflows/build-test-release.yml

```diff
@@ -557,3 +557,67 @@ jobs:
           artifacts: packages/*
           omitBody: true
           replacesArtifacts: true
+
+  publish-controller-image-marketplace-owned-ecr:
+    runs-on: ubuntu-latest
+    needs:
+      - validate-release-increment
+    if: ${{ needs.validate-release-increment.outputs.perform-release == 'true' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - name: Install tools
+        run: |
+          make install-tools
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Login to Amazon ECR Private
+        id: login-ecr-private
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Build and push container image to Amazon ECR
+        env:
+          CONTROLLER_IMG: 709825985650.dkr.ecr.us-east-1.amazonaws.com/lumigo/lumigo-kubernetes-operator:${{ needs.validate-release-increment.outputs.version }}
+        run: |
+          make VERSION=${{ needs.validate-release-increment.outputs.version }} docker-buildx-manager
+
+  publish-telemetry-proxy-image-marketplace-owned-ecr:
+    runs-on: ubuntu-latest
+    needs:
+      - validate-release-increment
+    if: ${{ needs.validate-release-increment.outputs.perform-release == 'true' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - name: Install tools
+        run: |
+          make install-tools
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Login to Amazon ECR Private
+        id: login-ecr-private
+        uses: aws-actions/amazon-ecr-login@v1
+      - name: Build and push container image to Amazon ECR
+        env:
+          PROXY_IMG: 709825985650.dkr.ecr.us-east-1.amazonaws.com/lumigo/lumigo-kubernetes-telemetry-proxy:${{ needs.validate-release-increment.outputs.version }}
+        run: |
+          make docker-buildx-telemetry-proxy
```

Comment:
Does the telemetry-proxy need a version for `make` like the previous job?

Reply:
im not sure (honestly I copied it from the current publish jobs that publish to the public ecr we have and they works so I think it is OK like this)